packages/access-api/migrations/0006_add_storage_provider_consumer_schema.sql

```diff
@@ -0,0 +1,21 @@
+-- Migration number: 0006 	 2023-03-02T16:40:04.407Z
+
+/*
+goal: add a table to keep track of the storage provider for a space.
+to enable https://github.com/web3-storage/w3protocol/issues/459
+*/
+
+CREATE TABLE
+    -- provision: the action of providing or supplying something for use
+    -- use case: representing the registration of a storage provider to a space
+    IF NOT EXISTS provisions (
+        id INTEGER NOT NULL PRIMARY KEY,
+        -- DID of the actor that is consuming the provider. e.g. a space DID
+        consumer TEXT NOT NULL,
+        -- DID of the provider e.g. a storage provider
+        provider TEXT NOT NULL,
+        -- DID of the actor that issued this provision
+        issuer TEXT NOT NULL,
```

Comment:
@Gozala The issuer isn't necessarily going to be billed, e.g. if there are several provisions for the same space. 

I don't want to attach any semantics to this other than 'the id of the thing that created this'. In ucan parlance that's `issuer` so that's why I used it. I think because when I used `author` you didn't like it, and I considered using [attributedTo](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-attributedto) but wanted to avoid the camelCase.

> It also may not be an issuer it's the with field that matters.

I can see how that might lead to confusion since the actor to attribute the capability to is not necessarily the final `issuer`.

> I think I would call it either account or subscriber

The thing is I tried to make this table generic enough to support this field not necessarily being an account did nor a subscriber...

> but something like sponsor or patron could also probably work.

I also considered `sponsor` so maybe we can agree there... I will update to use that.

Reply:
changed to sponsor https://github.com/web3-storage/w3protocol/pull/462/commits/754463da0a58ad02b42eac45091e0fda8780bd8c